.htaccess

```diff
@@ -14,6 +14,19 @@
     # RewriteBase /
 
     RewriteEngine On
+
+    # Deny access
+    RewriteRule "^config(?:.default)?\.php" "-" [NC,F]
+    RewriteRule "^web\.config" "-" [NC,F]
+    RewriteRule "^(?:cli|ChangeLog|LICENSE)" "-" [NC,F]
+
+    # Skip the deny rule for js, css and png
+    RewriteCond "%{REQUEST_URI}" "^/(?:venor|plugins)/.*\.(?:js|css|png)$"
+    RewriteRule ".?" "-" [S=1]
+
+    # Deny access to any non assets plugin files
+    RewriteRule "^(?:vendor|plugins)/" "-" [NC,F]
```

Comment:
Some plugins like the Calendar, Gantt or Chat contains assets. If do `curl http://example.org/plugins/Calendar/Assets/calendar.js` I got a 403.

Reply:
It's working for me:

```
curl -I https://kanboard.mouvement-democratie-reelle.fr/plugins/Calendar/Assets/calendar.js
HTTP/1.1 200 OK
Set-Cookie: cluster017BAK=R896099146; path=/; expires=Wed, 10-Jan-2018 18:40:54 GMT
Date: Wed, 10 Jan 2018 17:24:27 GMT
Content-Type: application/javascript
Content-Length: 2622
Set-Cookie: cluster017=R3592952301; path=/; expires=Wed, 10-Jan-2018 18:23:26 GMT
Server: Apache
Last-Modified: Sun, 07 Jan 2018 03:05:27 GMT
Accept-Ranges: bytes
Vary: Accept-Encoding
Cache-Control: max-age=900
Expires: Wed, 10 Jan 2018 17:39:28 GMT
X-IPLB-Instance: 17328
```

```
curl -I https://kanboard.mouvement-democratie-reelle.fr/plugins/Calendar/README.md
HTTP/1.1 403 Forbidden
Set-Cookie: cluster017BAK=R896102413; path=/; expires=Wed, 10-Jan-2018 18:40:01 GMT
Date: Wed, 10 Jan 2018 17:24:43 GMT
Content-Type: text/html; charset=iso-8859-1
Set-Cookie: cluster017=R3592952301; path=/; expires=Wed, 10-Jan-2018 18:34:04 GMT
Server: Apache
X-IPLB-Instance: 502
```

Reply:
I confess that my OVH hoster had some security activated and rejected the user agent of curl, so I disabled that security to make my tests.

Regards.